handbook.md

```diff
@@ -0,0 +1,233 @@
+
+**Max** is a training coordinator of an international research consortium. They recently surveyed researchers in the consortium on their need in HTS data analysis. The results were overwhelming: over 60 scientists on several locations would like to learn how to analyze their own RNA-seq data. To fulfill this demand, Max decided to organize a Gallantries event: a 3-days workshop with 4 different locations simultaneously (Greece, Estonia, France and Spain). Max is the **global organizer** of this event, in charge to find the date, contact and coordinate with the local hosts, advertize the event, find the instructors, etc. After the event, Max will aggregate all feedback from participants, helpers and hosts and share them.
+
+Max first contacted 4 **local hosts** (1 on each site) whose **Imani** in Greece. Imani is in charge to find a fitting room (preferably with computers), check the local setup, recruite local helpers, advertize the event locally for participants, organize the local catering and social event. Imani will also make the introduction and wrap-up each days, collect feedback from participants.
+
+Imani recruited 2 **local helpers** per day whose **Casey**. **Casey** has some previous experience with RNA-seq data analysis and Galaxy. Before the workshop, they went through the training material and tested it. They helps participants during the workshop when they are blocked or got different results than the instructors. They also gives direct feedback to the instructor about the pace, the possible local issues, etc.
+
+Max also recruited 4 **instructors** whose **Farah**. Farah is trained bioinformaticians and experienced instructors giving regularly training to scientists on HTS data analysis. They will teach the morning of the 2nd day (introduction and the frist steps of RNA-seq data analysis). Farah is located in Germany and teaches from a room at their institute in front of their computer. During this teaching morning, they adapt their pace given the feedback they got from the local helpers, check the status of participants job on a dedicated page and also answer questions from participants written on chat.
+
+**Alex** is a PhD student in molecular biology based in Greece. They would like to learn about RNA-seq data analysis to be able to analyze the data they generated. They heard about the workshop and join one site for the workshop close to their institute. As **participant**, Alex actively participates to the workshop by running their first RNA-seq data analysis given the instructor's instructions, asks for help to local helpers when stucks, raises their questions on the participant chat and gives feedback using sticky notes and dedicated feedback forms.
+
+# Curriculum and schedule for an event
+
+A typical Gallantries event covers the computational analysis of HTS data from the raw data to the development of publication ready visualizations of the analyzed results within **3 days** with hands-on training:
```

Comment:
Maybe "an example gallantries", rather than typical? So it isn't so restrictive to the HTS example, and doesn't give people the feeling they can't use it for other topics.